examples/mysql/docker-healthcheck

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Project: docker-library/healthcheck (https://github.com/docker-library/healthcheck)
+# File source: https://github.com/docker-library/healthcheck/blob/master/mysql/docker-healthcheck
+
+set -eo pipefail
+
+if [ "$MYSQL_RANDOM_ROOT_PASSWORD" ] && [ -z "$MYSQL_USER" ] && [ -z "$MYSQL_PASSWORD" ]; then
```

Comment:
MYSQL_USER and MYSQL_PASSWORD are not passed to the container...
Did you check its working?

Reply:
Yes, this file was taken from different repo (specified at the file header) and supports number of ways to start the MySQL container.
As this code is not mine I prefer not to change it.